generate.sh

```diff
@@ -29,49 +33,52 @@ INVENV=$(python -c 'import sys; print ("true" if hasattr(sys, "real_prefix") els
 # and change current directory to git repository.
 if [ "${WEBUI:-false}" == "true" ]; then
   BASE=$(pwd)
+  LOGFILE=${BASE}/temp/${EMAIL}/${UNIQUEID}.txt
   mkdir -p temp/${EMAIL} && cd $_
-  echo -e "Cloning Repository...\n"
+  echo -e "Cloning Repository...\n" | tee -a ${LOGFILE}
   git clone -q https://:@${PLATFORM}/${USERNAME}/${REPONAME} "$UNIQUEID" >/dev/null 2>&1
   if [ $? -ne 0 ]; then
     >&2 echo -e "Failed to Clone. Repository does not exist.\n"
     exit 4
   fi
   cd ${UNIQUEID}
-  echo -e "Repository Cloned Successfully!\n"
+  print_log "Repository Cloned Successfully!\n"
 else
   git clone -q https://github.com/fossasia/yaydoc.git yaydocclone
   BASE=$(pwd)/yaydocclone
 fi
 
+LOGFILE=${BASE}/temp/${EMAIL}/${UNIQUEID}.txt
```

Comment:
Does LOGFILE makes sense when running from travis? The else section would only run when running from travis